packages/cli/README.md

```diff
@@ -138,6 +138,11 @@ Commands are organized in a hierarchy that maps to the API hierarchy.
 * [`smartthings locations:rooms:delete [ID]`](#smartthings-locationsroomsdelete-id)
 * [`smartthings locations:rooms:update [ID]`](#smartthings-locationsroomsupdate-id)
 * [`smartthings locations:update [ID]`](#smartthings-locationsupdate-id)
+* [`smartthings plugins`](#smartthings-plugins)
+* [`smartthings plugins:install PLUGIN...`](#smartthings-pluginsinstall-plugin)
+* [`smartthings plugins:link PLUGIN`](#smartthings-pluginslink-plugin)
+* [`smartthings plugins:uninstall PLUGIN...`](#smartthings-pluginsuninstall-plugin)
+* [`smartthings plugins:update`](#smartthings-pluginsupdate)
```

Comment:
These are not part of the SmartThings API hierarchy like the note above suggests.
> Commands are organized in a hierarchy that maps to the API hierarchy.

Should we update that header verbiage a bit to make it clear some commands aren't hitting the API?

Reply:
What exactly makes this look like it's part of the API hierarchy? This text here is all auto-generated and the specify lines added here come from an external package so we would have to get them to change the wording or stop auto-generating this if we wanted to change it.

Reply:
The note above all of these commands explicitly states
> Commands are organized in a hierarchy that maps to the API hierarchy.

Yet, `plugins` and the associated subcommands are an oclif thing and not part of the SmartThings API. There are also other commands in this list that are not part of any SmartThings API. I just thought it might be confusing to some readers. I would not try to edit this section at all unless we could add delimiters or something but maybe state something above like 
> Commands **for SmartThings objects** are organized in a hierarchy that maps to the API hierarchy.

Otherwise what else is telling someone not super familiar with oclif that `smartthings plugins` isn't an API entity?

Reply:
I see what you're saying now. I've updated that text to:

Commands that use the SmartThings REST API are organized in a hierarchy
that maps to that hierarchy.